gqlalchemy/query_builder.py

```diff
@@ -330,14 +340,35 @@ def construct_query(self) -> str:
 
 
 class OrderByPartialQuery(PartialQuery):
-    def __init__(self, properties: str):
+    def __init__(self, properties: Union[str, Tuple[str, Order], List[Union[str, Tuple[str, Order]]]]):
         super().__init__(DeclarativeBaseTypes.ORDER_BY)
 
         self.properties = properties
 
     def construct_query(self) -> str:
         """Creates a ORDER BY statement Cypher partial query."""
-        return f" ORDER BY {self.properties} "
+
+        return f" {self.type} {(self._read_list(self.properties)) if isinstance(self.properties, list) else self._read_item(self.properties)} "
```

Comment:
This line is really long, [PEP8](https://peps.python.org/pep-0008/#maximum-line-length) says it should be up to 80 characters, but in our checks flake8 permits it, Why? Also, should we avoid such long lines for readability?

Reply:
Fixed